.github/workflows/linuxjs-flaky-tests.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     name: JS
     strategy:
       matrix:
-        group: [s-apputils]
+        group: [js-apputils, js-services]
```

Comment:
Nice correction on the js-apputils as well!

Reply:
Yes, looks like it was actually not running before. And seems to be failing quite often now.